<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>177</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Palatino Linotype</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Â¿Seguro que quieres BORRAR (nombre del evento) ?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBorrar">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    background-color: rgb(255, 168, 165); /* Un rojo pastel o rosa muy claro */
}</string>
    </property>
    <property name="text">
     <string>BORRAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCancelarEliminar">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>CANCELAR</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>495</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
